commands/Economy/Cooking/craft.js

```diff
@@ -1,75 +1,66 @@
-exports.run = async (client, msg, [item, amount]) => {
-    const sqlite3 = require("sqlite3").verbose();
-    let db = new sqlite3.Database("./assets/data/inventory.sqlite");
-    let items = require("../../../assets/values/items.json");
-    let recipe = require("../../../assets/values/recipes.json");
+const { Command } = require("klasa");
+const { MessageEmbed } = require("discord.js");
 
-    if (!item) { return msg.channel.send("You need to define an item to craft, baka!"); }
-    var product = items[item.toLowerCase()];
-
-    recipe = recipe[item.toLowerCase()];
-    if (!recipe) { return msg.channel.send("I couldn't find a recipe like that!"); }
+module.exports = class extends Command {
+    constructor(...args) {
+        super(...args, {
+            name: "craft",
+            enabled: true,
+            runIn: ["text"],
+            aliases: ["cook"],
+            cooldown: 30,
+            description: "Make items",
+            usage: "[item:str] [amount:str]", usageDelim: " "
+        });
+    }
 
-    if (amount === "help") {
-        var recipeList = [];
+    async run(msg, [item=item.toLowerCase(), amount=1]) {
+        const itemDB = this.client.itemData;
+        let tarItem = itemDB[item];
+        if (!tarItem) { return msg.channel.send(this.client.speech(msg, ["func-dataCheck", "noItems"])); }
+        if (!tarItem.recipe) { return msg.channel.send(this.client.speech(msg, ["craft", "noRecipe"])); }
 
-        recipe.forEach(element => { recipeList.push(element[1] + " " + items[element[0]].emote); });
+        if (amount === "help") {
+            var recipeList = [];
+            tarItem.recipe.forEach(element => { recipeList.push(`${element[1]} ${itemDB[element[0]].emote}`); });
+        
+            const embed = new MessageEmbed()
+                .setTitle(`${this.client.util.toTitleCase(tarItem.name)} ${tarItem.emote}`)
+                .addField(`Category: ${this.client.util.toTitleCase(tarItem.category)}`, true)
+                .addField("Price:", `Buy: ${(tarData.buy || "N/A")} - Sell: ${tarData.sell}`, true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['tarData' is not defined.](https://app.codacy.com/manual/Butterstroke/MargarineBot/pullRequest?prid=4918293)